packages/lib-panoptes-js/panoptes.js

```diff
@@ -0,0 +1,67 @@
+const superagent = require('superagent');
+const config = require('./config');
+
+function handleErrors(error) {
+  if (console) console.error(error);
+
+  return error;
+}
+
+// TODO: Write auth client
+function get(endpoint, query, host) {
+  const apiHost = host || config.host;
+  const request = superagent.get(`${apiHost}${endpoint}`)
+    .set('Content-Type', 'application/json')
```

Comment:
These get repeated a lot. Superagent [accepts an object](https://visionmedia.github.io/superagent/#setting-header-fields) to `.set()`

Reply:
I copied and pasted as a start from what I did for edu-api and I did this for a reason. I don't recall what that was though. I'll try the object again and see what happens via tests.